### DISCUSSION TITLE
Solution for two separate sites in one leihs system?

### DISCUSSION THREAD

**Discussion by bvontob-ringier** [leihs/leihs#1662] [Q&A / Community Support] [answered]
Hi all,

we're using leihs since many years to hand out video equipment to many teams – and are very happy with it! Now we're facing a new situation: We have to integrate (due to a merger) a completely different site (at the other end of town), with different teams, separate equipment, but obviously similar in nature (they e.g. have some of the same cameras and other equipment as our existing pool).

What we want to achive now (using buildings, pools, groups or any other means in leihs):
* People working at one site should primarily see the equipment there – so they don't accidentally book material from the other site (where they probably don't want to drive to pick it up – and we are not able to provide the service to ship the rentals around town). So they should not see a total of e.g. "5 cameras of type x available" if 2 are available in one site and 3 in the other, and they then think they can easily book 4 of them at once…
* We obviously don't want to create unnecessary duplicates of same equipment models for the two sites (if it's the same camera model, it should be only once in the database, and different instances assigned to different sites).
* We still want people to browse through the other site and, if they choose to do so, specifically book something from the other site (not their "home site") in special situations (e.g. something very specific is only available in one place, and they are willing to drive there to pick it up).

Is there an easy way to reach this goal with existing means available in leihs? We played around with quite some different ideas but none really gave us the above results…

Thx a lot, maybe somebody has an idea, or already implemented a similar scenario! (Or shares the same feature request, if it's not currently possible.)

Regards,
Beat

**Comment by neoheat** [comment, marked as answer]
Hi Beat

Nice to hear that you are using Leihs successfully! Please find my answers here:


the first two points you list are covered by existing features.

**Requirement 1:** Create a second inventory pool, authorize the users from each site to the according inventory pool.
**Requirement 2:** This is solved by means of the globally defined models. 

**Requirement 3:** There is no perfect solution. If a user has access to multiple pools, the items which are managed in separate pools are displayed combined in the borrow GUI. Currently, a user can filter for the desired pool (or in your wording site) in the modal window while checking the availability or adding an item into the basket. If a user reserves an item from the "secondary" pool, they could see the pickup location in the email (if you setup an according text to the confirmation mail).

<img width="913" alt="items in multiple parks - filter" src="https://github.com/leihs/leihs/assets/123965725/eca494ee-30a3-4416-85b9-2e40551d0e97">

however, we can't set primary or secondary pools - or even "read write" / "right to request" on inventory pools.

I hope this is a possible solution for you.
Best, Lukas

**Reply by bvontob-ringier** [reply]
Thx a lot for the answer/ideas! Yes, getting requirement 1 plus 3 solved still is a problem. We fear that a lot of users will accidentally book material from the wrong site – but we'll try. And/or maybe once there will even be a corresponding feature in leihs that makes it possible… ;) Thx anyway!
